email_tracking/sample.env

```diff
@@ -0,0 +1,2 @@
+EASYPOST_API_KEY=(get from https://www.easypost.com/account#/api-keys)
+SENDGRID_API_KEY=(get from https://app.sendgrid.com/settings/api_keys - if it doesnt already exist create one by clicking on create api key button on top right cornor)
```

Comment:
I'd probably just leave this blank and keep the instructions on getting them in the README.